lib/bitcrowd_ecto/changeset.ex

```diff
@@ -316,4 +317,106 @@ defmodule BitcrowdEcto.Changeset do
       changeset
     end
   end
+
+  @doc "Validates the currency of the given money field"
+  @doc since: "0.13.0"
+  @spec validate_currency(Changeset.t(), atom(), atom()) :: Changeset.t()
+  def validate_currency(changeset, field, required_currency \\ :EUR) do
+    validate_change(
+      changeset,
+      field,
+      :valid_currency,
+      fn _, %Money{currency: currency} ->
+        if currency == required_currency do
+          []
+        else
+          [
+            {field, {"invalid currency", [validation: :validate_currency, fields: [field]]}}
+          ]
+        end
+      end
+    )
+  end
+
+  @doc "Validates amount of money is greater than"
+  @doc since: "0.13.0"
+  @spec validate_more_money(Changeset.t(), atom(), Money.t()) :: Changeset.t()
+  def validate_more_money(changeset, field, money) do
```

Comment:
how about calling `money` `target` or so?

Reply:
I don't have a better name than `money` but `target` also doesn't feel descriptive enough.

Reply:
Do you have another idea?

Reply:
Thinking about this again, I really don't have a strong opinion about the name we use here, so I will use `target`.